jvb/src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -161,7 +161,9 @@ public void setVideoType(String sourceName, VideoType videoType) {
         }
         else
         {
-            logger.error("setVideoType - source description not found for: " + sourceName);
+            // Logging on info level because this is expected to happen if the client opens the data channel, before
+            // the sources are signaled (see lib-jitsi-meet).
+            logger.info("setVideoType - source description not found for: " + sourceName);
```

Comment:
Does the client need to retransmit?

Reply:
No, but only because we know how lib-jitsi-meet works. This message is missed for the camera, because there's no start with screensharing feature anymore. Otherwise the screen video type never arrives before sources. It can be flaky but I don't want to add caching given that things will change with source maps.